kedro/framework/cli/starters.py

```diff
@@ -23,20 +24,29 @@
     _clean_pycache,
     _filter_deprecation_warnings,
     command_with_verbosity,
+    load_entry_points,
 )
 
 KEDRO_PATH = Path(kedro.__file__).parent
 TEMPLATE_PATH = KEDRO_PATH / "templates" / "project"
 
+_STARTERS_REPO = "git+https://github.com/kedro-org/kedro-starters.git"
+
+# The `astro-iris` was renamed to `astro-airflow-iris`, but old (external) documentation
+# and tutorials still refer to `astro-iris`. We create an alias to check if a user has
+# entered old `astro-iris` as the starter name and changes it to `astro-airflow-iris`.
 _STARTER_ALIASES = {
-    "astro-airflow-iris",
-    "mini-kedro",
-    "pandas-iris",
-    "pyspark",
-    "pyspark-iris",
-    "spaceflights",
+    "astro-airflow-iris": {"repo": _STARTERS_REPO, "directory": "astro-airflow-iris"},
+    "astro-iris": {
+        "repo": _STARTERS_REPO,
+        "directory": "astro-airflow-iris",
+    },  # this is an alias name
+    "mini-kedro": {"repo": _STARTERS_REPO, "directory": "mini-kedro"},
+    "pandas-iris": {"repo": _STARTERS_REPO, "directory": "pandas-iris"},
+    "pyspark": {"repo": _STARTERS_REPO, "directory": "pyspark"},
+    "pyspark-iris": {"repo": _STARTERS_REPO, "directory": "pyspark-iris"},
+    "spaceflights": {"repo": _STARTERS_REPO, "directory": "spaceflights"},
```

Comment:
Let's make this `template_path` so it matches the arguments given in `new`. This also keeps it generic.

Reply:
Done!